mail/views.py

```diff
@@ -52,3 +54,34 @@ def post(self, request, *args, **kargs):  # pylint: disable=unused-argument, no-
                 } for batch_num, resp in enumerate(mailgun_responses)
             }
         )
+
+
+class FinancialAidMailView(APIView):
+    """
+    View for sending financial aid emails to individual learners
+    """
+    authentication_classes = (authentication.SessionAuthentication, )
+    permission_classes = (permissions.IsAuthenticated, UserCanMessageLearnersPermission, UserCanEditFinancialAid)
+
+    def post(self, request, *args, **kwargs):
+        """
+        Post request to send emails to an individual learner
+        """
+        try:
+            subject = request.data.get('email_subject') or kwargs['email_subject']
```

Comment:
@gsidebo felt strongly about using the views for routing all mailgun communications in our conversation on HipChat ("lots and lots of reasons to funnel all of our communication with the mailgun API through views like the one thats already there").  So this allows us to call `FinancialAidMailView().post()` from within another view to also send emails.

Gavin, I would love your input here as well.  I'm happy to go either way.